src/ansys/mapdl/core/mapdl_core.py

```diff
@@ -434,7 +435,7 @@ def components(self) -> "ComponentManager":
 
         >>> mapdl.solution.converged
         """
-        if self._exited:  # pragma: no cover
+        if self.exited:  # pragma: no cover
```

Comment:
The class seem to be mixing `self.exited` and `self._exited`. Are you sure about the current behavior ?

Reply:
You are right. I should use the API. Thank you @SMoraisAnsys !